test/integration_test.go

```diff
@@ -1170,6 +1171,19 @@ func TestAutoRetryExhaustRetries(t *testing.T) {
 	assert.NotContains(t, out.String(), "Apply complete!")
 }
 
+func TestAutoRetryCustomRetryableErrors(t *testing.T) {
```

Comment:
Prob worth adding another test that uses the same module, but does _not_ set retryable errors to make sure that test fails!

Reply:
Addressed everything but this one - added a TODO as discussed on Slack

Reply:
This is now addressed in 7dfdc8c.